styx-scheduler-service/src/test/java/com/spotify/styx/StateInitializingTriggerTest.java

```diff
@@ -150,12 +185,26 @@ public void shouldCreateWorkflowInstanceParameter() throws Exception {
     }
   }
 
-  private WorkflowConfiguration workflowConfiguration(Schedule schedule, String... args) {
+  private static WorkflowConfiguration workflowConfiguration(Schedule schedule, String... args) {
     return WorkflowConfiguration.builder()
         .id("styx.TestEndpoint")
         .schedule(schedule)
         .dockerImage("busybox")
         .dockerArgs(ImmutableList.copyOf(args))
         .build();
   }
+
+  @Test
+  public void shouldWrapIsClosedExceptionIntoRuntime() throws Exception {
```

Comment:
what are we testing in this test? is it something we added in this commit?

Reply:
No it was a hole is our test coverage